client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.js

```diff
@@ -81,23 +96,37 @@ const DataFilesSearchbar = ({
           type="search"
           minLength="3"
           onChange={onChange}
-          value={query}
+          value={query || ''}
           name="query"
-          aria-label={`Search (in ${sectionName})`}
+          aria-label={`Search`}
           styleName="input"
           className="form-control"
-          placeholder={`Search in ${sectionName}`}
+          placeholder={`Search`}
           data-testid="input"
           autoComplete="off"
           disabled={disabled}
         />
-        {hasQuery && (
+        {hasQuery && !siteSearch && (
           <div aria-label="Summary of Search Results" styleName="results">
             {resultCount} Results Found for <span>{hasQuery}</span>
```

Comment:
When we have two files with the same name in a system (e.g. `frontera` [`Folder`] and `frontera.py` [`Code`]), searching for **frontera** and then setting the `File Type` for a `Folder` still results in a `resultCount` of 2 results for **frontera** even though only `frontera` [`Folder`] is showing in the listing.

Is this intended behavior?

Reply:
Good question! I'll ask design

Reply:
@duckonomy This is fixed!